### DISCUSSION TITLE
php is not recognized with process component

### DISCUSSION THREAD

**Discussion by PranitaEncureIT** [symfony/symfony#50229] [Symfony Book: The Fast Track]
I tried this:
$process = Process::fromShellCommandline('php artisan serve', base_path());
$process->setTimeout($timeout);
$process->mustRun();
$output = $process->getOutput();

But I got this error:
'php' is not recognized as an internal or external command, operable program or batch file.

**Comment by George-Gabro** [comment]
It seems like PHP is not installed on your machine,
Try checking if PHP is installed first, then check environment variables if both didn't work you may use PHP binary file that is actually "not installed" on your system if that's so try specifying the path like this '$process = Process::fromShellCommandline('./vendor/bin/php artisan serve', base_path());' or wherever your php binary file is

**Reply by PranitaEncureIT** [reply]
Thank you for your reply.
It does related to the path.
